.appveyor.yml

```diff
@@ -87,6 +87,8 @@ for:
         export CROSS_TOOLS=/usr/local/src/cross
         export WASM=OFF
         export LLVM_TAR=llvm-17.0.6-ubuntu18.04-Release+Asserts-x86.arm.wasm.tar.xz
+        export XWIN_HOME=$APPVEYOR_BUILD_FOLDER/xwin
```

Comment:
I like the approach with `xwin`.

But I'm concerned that it's a personal project, which we supposed to download from personal github releases of project and use in the CI. The project is nice and it's not too hard to review it to make sure that it doesn't pose a security threat. But there's no mechanism to verify that it's not modified (even if we use a fixed version), unless we download a SHA256 and keep in our secrets (which is also a possible mitigation).

What would solve this problem is to distribute `xwin` through one of the package managers. Do you know if it's available through one of them or is it possible to request that?

Reply:
Yeah that's definitely a valid point, I've not got the answer to this one except for perhaps using cargo directly at this point. Maybe we can get @Jake-Shadle into the conversation about this one as well.

Reply:
I never package projects I maintain, if you don't trust the github releases you can use `cargo install` I suppose to build it yourself before running it, though obviously at the cost of a lot of CPU work.

Reply:
@Jake-Shadle Thanks for commenting on that and thanks for the nice tool.

This is the code that runs in trusted environment, i.e. has access to github secrets, affects what is linked in to the binaries we ship, etc. So it requires some procedures in place to ensure security and integrity of the result. And operating in enterprise environment requires taking these kind of things more serious.

So I guess we can cache the SHA256 of the package in out secrets and use the file from github releases. That's would be the easiest for everyone.

But that would really, really help adopting `xwin` if it's available through some package manager. `apt`, `snap`, or whatever.

Reply:
IMO it would be easiest to just create a tarball or container image with the CRT/SDK in it that you can download in your CI, they don't change frequently and it means you don't run any code at all.

Reply:
If we run it in public CI, then the image needs to be in public, this implies that we are responsible for it and we need to review with security people, maintain, etc. And this is something we definitely don't want to do.

`xwin` seems to be ideal tool for that purpose. If it would only be shipped though some package manager... ;-)